index.html

```diff
@@ -38,11 +38,26 @@
 				</div>
 				<a class="header__menu-icon">
 					<object
+						class="header__menu-icon-svg"
 						type="image/svg+xml"
 						data="/images/light/menu_icon_light.svg"
 					></object>
 				</a>
 			</div>
+			<div class="menu__slider">
+				<a class="menu__button-close">
+					<object
+						class="menu__button-svg"
```

Comment:
Don't use svg in your class name as this may change later

Reply:
Done